extras/tmux/README.md

```diff
@@ -2,3 +2,7 @@
 1. clone this repository `git clone https://github.com/comfysage/evergarden.git`
 2. copy tmux directory to tmux plugins `cp -R ./extra/tmux/* ~/.tmux/plugins/evergarden-tmux`
 3. add `run ~/.tmux/plugins/evergarden-tmux//evergarden.tmux` to your `.tmux.conf`
+
+Shoutout to
+  - [catppuccin/tmux](https://github.com/catppuccin/tmux)
+  - [tao of tmux](https://tao-of-tmux.readthedocs.io/en/latest/manuscript/09-status-bar.html)
```

Comment:
i understand that `tao of tmux` has probably been a useful resource for you to create this status bar but i see no reason to add this here.

good thinking to mention the catppuccin repo. it seems like youve based your work on them so its smart to refer to them.

Reply:
Removed `tao of tmux`